code/controllers/subsystem/SSstatpanel.dm

```diff
@@ -137,23 +136,40 @@ SUBSYSTEM_DEF(statpanels)
 		return list()
 
 	var/datum/object_window_info/obj_window = load_from.obj_window
+	if(!obj_window)
+		return list()
 	var/list/already_seen = obj_window.atoms_to_images
 	var/list/to_make = obj_window.atoms_to_imagify
 	var/list/turf_items = list()
+	var/i = 0
 	for(var/atom/turf_item as anything in obj_window.atoms_to_show)
+		// Limit what we send to the client's rendered section.
+		if(i < min_index)
+			i++
+			continue
+		if(i >= max_index)
+			i++
+			continue
+		i++
```

Comment:
I believe this is logically equivalent, and should give us super worthwhile bytecode improvements
```suggestion
		i++
		if(i <= min_index || i > max_index)
			continue
```